content/cloud-computing/cloud-computing.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Cloud Computing'
+Description: 'Cloud computing refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cloud-computing'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Cloud computing** refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user. This is becoming popular with businesses and ubiquitous in personal computing. For example, whenever a person sets up a website on GoDaddy, plays World of Warcraft, or streams a movie on Netflix, they are using a cloud computing solution.
+
+## Advantages
+
+Cloud computing offers the following advantages to the user:
+
+- **Scalability:** Adding additional capacity is quicker and easier in a cloud environment then in an on-premises data center. When a user manages a data center themselves, additional capacity involves purchasing, installing and configuring new hardware. A cloud solution provides access to many more resources than would otherwise be possible.
```

Comment:
```suggestion
- **Scalability:** Adding additional capacity is quicker and easier in a cloud environment then in an on-premises data center. When a user manages a data center themselves, additional capacity involves purchasing, installing, and configuring new hardware. A cloud solution provides access to many more resources than would otherwise be possible.
```